tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
 
-envlist = py27,py35,py36
+envlist = py{27,35,36}
 
 [testenv]
 commands=
-    pip install --global-option=build_ext --global-option=--gdal-config=/usr/bin/gdal-config --global-option=-I/usr/include/gdal GDAL=={env:GDAL_VERSION:2.3.1}
+    pip install --global-option=build_ext --global-option=--gdal-config=/usr/bin/gdal-config --global-option=-I/usr/include/gdal GDAL=={env:GDAL_VERSION:2.1.3}
```

Comment:
@pauricthelodger - rolled bak to GDAL 2.1.3. I am running into some issues installing the other versions in the test env. Might update that later.